rubicon/java/api.py

```diff
@@ -170,6 +184,15 @@ def select_polymorph(polymorphs, args):
                     b"Ljava/lang/CharSequence;",
                     b"Ljava/lang/Object;",
                 ])
+            elif isinstance(arg, Sequence) and len(arg) > 0:
+                # If arg is an iterable of all the same basic numeric type, then
+                # an array of that Java type can work.
+                if isinstance(arg[0], (int, jint)):
+                    if all((isinstance(item, (int, jint)) for item in arg)):
+                        arg_types.append([b'[I'])
+                elif isinstance(arg[0], (float, jfloat, jdouble)):
+                    if all((isinstance(item, (float, jfloat, jdouble)) for item in arg)):
+                        arg_types.append([b'[D', b'[F'])
```

Comment:
What's the fallback case here? If `arg` is a sequence of non-zero length, and types *aren't* consistent, this won't throw an unknown argument error (or any other type of error) - it will move on to trying to create an option

Reply:
This would also seem to rule out passing in an empty array under any circumstances...

Reply:
Both should be clarified in a fresh push